apm-lambda-extension/main_test.go

```diff
@@ -0,0 +1,343 @@
+package main
+
+import (
+	"bytes"
+	e2e_testing "elastic/apm-lambda-extension/e2e-testing"
+	"elastic/apm-lambda-extension/extension"
+	"elastic/apm-lambda-extension/logsapi"
+	json "encoding/json"
+	"github.com/google/uuid"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+)
+
+type RegistrationResponse struct {
```

Comment:
We already have a struct for this extension.RegisterResponse, maybe you can reuse it?

Reply:
Solved in [d9a8edc](https://github.com/elastic/apm-aws-lambda/pull/136/commits/d9a8edc0ca29cd924a0217ed22f9715a5c270c90)